tiledb/sm/filesystem/azure.cc

```diff
@@ -342,15 +335,12 @@ Status Azure::flush_blob(const URI& uri) {
     //
     // Alternatively, we could do nothing and let Azure release the
     // uncommited blocks ~7 days later. We chose to delete the blob
-    // as a best-effort operation. We intentionally are ignoring the
-    // returned Status from 'remove_blob'.
```

Comment:
Ignoring the error of `remove_blob` makes sense (it's unfortunate that documentation does not match with the code). You could do it now if you want, or please open a story if you don't want change existing behavior.